test/unit/components/report.unit.js

```diff
@@ -347,7 +347,7 @@ describe('Report', function() {
       send_request = sinon.stub(utils, 'send_request');
     });
 
-    it('should prepend "custom/" to any numerical id', function(done) {
+    xit('should prepend "custom/" to any numerical id', function(done) {
```

Comment:
Could you elaborate on why this test is now skipped?

Reply:
I found the issue. At the end of the string on line 356, the `.+` fails because there are no longer any query parameters. This should be changed to `.*`. If you make that change I'll approve